package/appimage/flameshot.AppDir/flameshot.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
```

Comment:
Flameshot already maintains desktop files on the ./docs/desktopEntry directory, why duplicate the efforts?